lib/src/over_react_redux/devtools/middleware.dart

```diff
@@ -0,0 +1,158 @@
+@JS()
+library over_react_redux.devtools.middleware;
+
+import 'dart:convert';
+import 'dart:html';
+import 'dart:js_util';
+
+import 'package:logging/logging.dart';
+import 'package:react/react_client/js_backed_map.dart';
+import 'package:redux/redux.dart';
+import 'package:redux_dev_tools/redux_dev_tools.dart';
+import 'package:js/js.dart';
+
+@JS()
+@anonymous
+class ReduxDevToolsExtensionConnection {
+  external factory ReduxDevToolsExtensionConnection();
+  external subscribe(dynamic Function(JsMap data) listener);
+  external send([dynamic action, dynamic state]);
+  external init([dynamic initialState]);
+}
+
+@JS('__REDUX_DEVTOOLS_EXTENSION__.connect')
+external ReduxDevToolsExtensionConnection reduxExtConnect([dynamic options]);
+
+class OverReactReduxDevToolsMiddleware extends MiddlewareClass {
+  Store _store;
+  ReduxDevToolsExtensionConnection devToolsExt;
+  final Logger log = new Logger('OverReactReduxDevToolsMiddleware');
+
+  OverReactReduxDevToolsMiddleware() {
+    log.onRecord.listen((LogRecord rec) {
```

Comment:
#nit Dart 2 also doesn't like when you type function expression parameters  ¯\_(ツ)_/¯
```suggestion
    log.onRecord.listen((rec) {
```